saleor/checkout/tests/test_calculations.py

```diff
@@ -67,6 +71,23 @@ def tax_data(checkout_with_items, checkout_lines):
     )
 
 
+@pytest.fixture
+def checkout_with_item_and_gift_card(checkout_with_item, gift_card):
+    # Add the gift card to the checkout
+    checkout = checkout_with_item
+    checkout.gift_cards.add(gift_card)
+    checkout.save()
```

Comment:
No need to make `.save` here. `gift_cards` are many to many relations. `.add` is enough :)